tests/com/carolinarollergirls/scoreboard/core/impl/ScoreboardImplTests.java

```diff
@@ -1530,15 +1555,15 @@ public void testPeriodJamInsertRemove() {
         assertEquals(4, p1.getAll(Period.NChild.JAM).size());
         assertEquals(1, p1.getMinNumber(Period.NChild.JAM) + 0);
         assertEquals(4, p1.getMaxNumber(Period.NChild.JAM) + 0);
-        assertEquals(1, sb.getAll(Period.NChild.JAM).size());
-        assertEquals(5, sb.getMinNumber(Period.NChild.JAM) + 0);
+        assertEquals(5, sb.getAll(Period.NChild.JAM).size());
```

Comment:
Another red flag: This should not contain anything but the upcoming jam.

Reply:
Why would we only have 1 jam in a period when we're on jam 5 of that period?

Reply:
sb is not a period but the scoreboard object.